org-ros.el

```diff
@@ -43,24 +43,47 @@
 (defcustom org-ros-secondary-screencapture-switch "-s"
   "Secondary screencapture software selection switch."
   :type 'string)
- 
+
+(defcustom org-ros-powershell-screencapture "powershell.exe"
+  "Powershell screencapture software (for people using wsl2)."
```

Comment:
@ahbc-sdg can you please change this line too to refer native windows and wsl?

With this done we will wait for MELPA to update it, Idn how it will react to the extra ps1 file.

Reply:
will do